### PR TITLE
Make the manpage more clear

### PR DESCRIPTION
**Pull request by N-R-K**
some of the terms used in the manpage aren't exactly obvious, especially
to a new user. this is an attempt at making the manual more clear and
remove any potential confusion.

here's a couple issues i've noticed where users were confused due to non
clear manpage description:

https://github.com/jarun/nnn/issues/1268
https://github.com/jarun/nnn/issues/1269
https://github.com/jarun/nnn/issues/1209

### CODE REVIEW

**Comment by N-R-K** [comment]
I'm not entirely sure what `+` is supposed to do. It doesn't seem to do anything for me.

**Comment by PatrickF1** [comment]
> I'm not entirely sure what + is supposed to do. It doesn't seem to do anything for me.

😂 I just spent another 30 minutes experimenting and still couldn't figure out what it does. I'm tempted to learn C just to find out what it does.

File: nnn.1 [hunk 1]
```diff
@@ -142,8 +144,8 @@ supports the following options:
         show version and exit
 .Pp
 .Fl x
-        show notis on selection cp, mv, rm completion
-        copy path to system clipboard on select
+        show notis on selection cp, mv, rm completion (requires \fI.cbcp\fR plugin)
```
Comment (PatrickF1) [inline nnn.1:147]: Notis = notification right? Why not just type out the whole thing?

**Comment by N-R-K** [comment]
It's used here

https://github.com/jarun/nnn/blob/bd6e7e81479823fdf28cdc507027982011039c9f/src/nnn.c#L7036-L7038

Which suggests it's about advancing the cursor after opening something, but it doesn't seem to do anything for me.

File: nnn.1 [hunk 2]
```diff
@@ -144,8 +144,8 @@ supports the following options:
         show version and exit
 .Pp
 .Fl x
-        show notis on selection cp, mv, rm completion (requires \fI.cbcp\fR plugin)
-        copy path to system clipboard on select (requires \fI.ntfy\fR plugin)
+        show notifications on selection cp, mv, rm completion (requires \fI.ntfy\fR plugin)
+        copy path to system clipboard on select (requires \fI.cbcp\fR plugin)
```
Comment (jarun) [inline nnn.1:148]: ... to system clipboard on selection

**Review by jarun — CHANGES_REQUESTED** [review]

File: nnn.1 (hunk 2 above)
Comment (N-R-K) [reply nnn.1:148]: How about
> copy all selection path to system clipboard

Comment (jarun) [reply nnn.1:148]: No it does each time when you select a single file also.

**Comment by jarun** [comment]
Toggle auto-advance -> please make this Toggle auto-proceed on file open

File: nnn.1 [hunk 3]
```diff
@@ -89,6 +89,7 @@ supports the following options:
 .Pp
 .Fl J
         disable auto-proceed on select
```
Comment (jarun) [inline nnn.1:91]: disable auto-proceed on select

> disable auto-proceed on selection

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
`-A` documentation: please change auto-select to auto-enter

**Comment by jarun** [comment]
Please ensure the word `advance` is gone from the manpage.

File: nnn.1 [hunk 4]
```diff
@@ -23,7 +23,7 @@ is a performance-optimized, feature-packed fork of noice
 .Em http://git.2f30.org/noice/
 with seamless desktop integration, simplified navigation,
 .Em type-to-nav
-mode with auto select, disk usage analyzer mode, bookmarks,
+mode with auto enter, disk usage analyzer mode, bookmarks,
```
Comment (jarun) [inline nnn.1:26]: dir auto-enter


Use the hyphen

File: nnn.1 [hunk 5]
```diff
@@ -88,7 +88,8 @@ supports the following options:
         show current file information in info bar (may be slow)
 .Pp
 .Fl J
-        disable auto-proceed on select
+        disable auto-proceed on selection
+        (eg. making a selection will no longer move cursor to the next entry)
```
Comment (jarun) [inline nnn.1:92]: (eg. making a selection

>

(e.g. selecting an entry...

**Review by jarun — CHANGES_REQUESTED** [review]

**Comment by jarun** [comment]
I think we are good. Please enable the merge option.

**Comment by N-R-K** [comment]
Looks like we got conflict. I'll rebase on top of master.

**Comment by jarun** [comment]
Thank you!

**Comment by PatrickF1** [comment]
Thank you @N-R-K! I appreciate you hoping on this so quickly after I opened up my issues in confusion. 
I still have some ideas. Is the ToDo issue a good place to discuss?

**Comment by N-R-K** [comment]
> I still have some ideas. Is the ToDo issue a good place to discuss?

@PatrickF1 Yes, the ToDo issue is a good place for small discussions.

**Pull request merged**